tests/kernel/kirk.pm

```diff
@@ -13,12 +13,16 @@ use strict;
 use testapi;
 use warnings;
 use serial_terminal 'select_serial_terminal';
+use version_utils qw(is_transactional);
+use transactional 'trup_install';
+use LTP::utils;
 
 our $result_file = 'result.json';
 
 sub run
 {
     my ($self) = @_;
+    my $install_from = get_var('KIRK_FROM', 'repo');
```

Comment:
Shouldn't we use same logic like with LTP installation?

We have `INSTALL_LTP`  variable which can contain `from_repo`. Maybe `INSTALL_KIRK` would be more compatible?

Reply:
I try to avoid any module specific variable that starts with a generic tag. For example, LTP module should contain variables starting with `LTP_` tag. Same story for `KIRK_` or any other module.
In this way it's easier to understand and to find variables used by specific modules.

I would say that `INSTALL_LTP` is not a good name and probably `LTP_INSTALL` would be better. I could switch from `KIRK_FROM` to `KIRK_INSTALL`, eventually.

Reply:
Renamed to `KIRK_INSTALL`